builders/manage/sudo/upgrade.md

```diff
@@ -13,25 +13,51 @@ In this guide, you'll learn how to use Sudo to upgrade your Tanssi appchain's ru
 
 ## Checking Prerequisites {: #checking-prerequisites }
 
-For the examples in this guide, you will need to have the following:
+For the example in this guide, you will need to have the following:
 
  - A Tanssi appchain (Snap or Dedicated)
  - Your appchain's Sudo account connected to your appchain's Polkadot.js Apps. You can refer to the [Managing Sudo guide](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank} for instructions on injecting your Sudo account into Polkadot.js Apps
+- The new [Wasm runtime](/learn/framework/architecture/#runtime){target=\_blank} binary file, built with a higher version than the current one
 
-If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+If you're unsure what your Tanssi appchain's Sudo account or runtime version is, you can find them in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
 
-![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/upgrade/upgrade-1.webp)
+![Locating your Sudo address and runtime version on apps.tanssi.network](/images/builders/manage/sudo/upgrade/upgrade-1.webp)
 
 !!! warning
     You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
 
+## Obtaining the Wasm Runtime {: #obtaining-wasm-runtime }
+
+If your chain is based on one of the official templates, you can download the official Wasm runtime binary file from the table below. The official releases are published in the [releases section](https://github.com/moondance-labs/tanssi/releases){target\_blank} in the Tanssi repository.
+
+|      Version       |    EVM Template    | Substrate Template |
+|:------------------:|:------------------:|:------------------:|
```

Comment:
Looks like this table can possibly be formatted. Not sure if you have the vscode extension: https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter